### PR TITLE
th-337: Remove time field from order form

### PR DESCRIPTION
**Pull request by DmytroVoronkov**
- removed time field
- fixed form size
- added scroll to form
- set 50/50 ratio to page
- fixed validation errors

image

### CODE REVIEW

**Review by h0wter — APPROVED** [review]

File: frontend/src/pages/order/libs/components/styles.module.scss [hunk 1]
```diff
@@ -1,28 +1,14 @@
 @import "src/assets/css/vars.scss";
 
 .formWrapper {
+  flex-grow: 1;
   width: 100%;
-  max-width: 700px;
-  height: calc(100vh - 271px);
-  padding: 20px 100px 35px;
+  padding: 20px 15% 35px;
```
Comment (kalinkaaaa14) [inline frontend/src/pages/order/libs/components/styles.module.scss]: can we write padding in px?

File: frontend/src/pages/order/libs/components/libs/helpers.ts [hunk 2]
```diff
@@ -2,13 +2,19 @@ import { type OrderCreateRequestDto } from '~/packages/orders/orders.js';
 
 import { TruckCarsQuantity } from './enums.js';
 
+const getCurrentDate = (): string => {
+  const currentDatetime = new Date();
+
+  return currentDatetime.toISOString().slice(0, 16);
```
Comment (kalinkaaaa14) [inline frontend/src/pages/order/libs/components/libs/helpers.ts]: why do you need to use slice? 
add a separate helper for it

**Comment by kalinkaaaa14** [comment]
as we decided that we will be considering all orders = asap orders then actually scheduled time is not needed at our db. 
but it the future we can implement planning future orders, so ok, let's leave this column in order table

**but scheduled time it's better to calculate on be, so remove it from the default values on fe, and add to service layer calculation of scheduled time field.**

**Review by h0wter — APPROVED** [review]

File: shared/src/packages/orders/libs/enums/orders-validation-message.enum.ts [hunk 3]
```diff
@@ -5,6 +5,9 @@ const OrdersValidationMessage = {
   DRIVER_ID_MUST_BE_NUMBER: 'Orders driverId must be number',
   TRUCK_ID_MUST_BE_NUMBER: 'Orders truckId must be number',
   CARS_QTY_MUST_BE_NUMBER: 'Orders carsQty must be number',
+  FIELD_IS_REQUIRED: 'This field is mandatory',
+  NAME_NOT_VALID: 'Must be 1 - 40 characters, start with a letter, no digits',
+  PHONE_NOT_VALID: 'Must be 7-18 digits, start with +',
```
Comment (Druxight) [inline shared/src/packages/orders/libs/enums/orders-validation-message.enum.ts]: Same messages but different range format. 
```suggestion
  NAME_NOT_VALID: 'Must be 1 - 40 characters, start with a letter, no digits',
  PHONE_NOT_VALID: 'Must be 7 - 18 digits, start with +',
```

**Review by Druxight — CHANGES_REQUESTED** [review]

**Review by AlexVlaso — APPROVED** [review]

**Review by denred — APPROVED** [review]

**Pull request merged**